roles/etcd_migrate/tasks/add_ttls.yml

```diff
@@ -0,0 +1,36 @@
+---
+# To be executed on first master
+- delegate_to: "{{ groups.oo_first_master[0] }}"
```

Comment:
Using `delegate_to` is not a good practise. That decision should be made on a play level.